packages/transport-native/package.json

```
@@ -0,0 +1,79 @@
+{
+    "name": "trezor-transport-native",
```

```suggestion
    "name": "@trezor/transport-native",
```
shoudn't it be consistent with the rest of the packages?

---

fixed